README.md

```diff
@@ -149,154 +142,112 @@ class MyClass {
         id int primary key,
         name text
       )
-    `)
+    `);
   }
 }
 ```
 
-This will output the template with margins stripped:
+This outputs the sensible:
 
 ```sql
 create table student(
-  id int primary key,
+  id int primary key
   name text
 )
 ```
 
-Custom expressions would work without any special composition of tag template functions:
+Expressions can be directly supported, as well as composition with tagged
+template functions:
 
-<pre lang="javascript">
+```javascript
 const query = sql@`
   select *
   from studients
   where name = ${name}
-`
-</pre>
-
-The behavior when later lines lack the whitespace prefix of the first line, is not yet defined:
-
-<pre lang="javascript">
-const tbd = @`
-  The first line starts with two spaces
-but a later line doesn't.
-`
-</pre>
+`;
+```
 
 ## In other languages
 
-- *Java* - [text blocks](https://openjdk.java.net/jeps/378) using triple-quotes
-- *Scala* - [multiline strings](https://docs.scala-lang.org/overviews/scala-book/two-notes-about-strings.html) using triple-quotes and `.stripMargin`
-- *Python* - [multiline strings](https://docs.python.org/3/library/textwrap.html) using triple-quotes to avoid escaping and `textwrap.dedent`
-- *Jsonnet* - [text blocks](https://jsonnet.org/learning/tutorial.html) with `|||` as a delimiter
-- *Bash* - [`<<-` Heredocs](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_07_04)
-- *Ruby* - [`<<~` Heredocs](https://www.rubyguides.com/2018/11/ruby-heredoc/)
-- *Swift* - [multiline string literals](https://docs.swift.org/swift-book/LanguageGuide/StringsAndCharacters.html#ID286) using triple-quotes - strips margin based on whitespace before closing delimiter
-* _PHP_  - `<<<` [heredoc/nowdoc](https://wiki.php.net/rfc/flexible_heredoc_nowdoc_syntaxes#closing_marker_indentation)  The indentation of the closing marker dictates the amount of whitespace to strip from each line.
+- *Java* - [text blocks](https://openjdk.java.net/jeps/378) using triple-quotes.
+- *Scala* - [multiline strings](https://docs.scala-lang.org/overviews/scala-book/two-notes-about-strings.html)
+  using triple-quotes and `.stripMargin`.
```

Comment:
A couple more newlines in the middle of these bullets

Reply:
The newlines are just for line wrapping. Because these lines don't start with a `-` list marker, they'll be joined with the preceding line like a paragraph.